### PR TITLE
Enable LED indicator functionality

### PR DESCRIPTION
**Pull request by lucasteligioridis**
This includes functionality to adjust the LED behaviour for the fan switch as well as modifying the orientation of which way the LED will point.

This closes: https://github.com/Koenkk/zigbee2mqtt/issues/18620

Pending merge of: https://github.com/Koenkk/zigbee-herdsman/pull/937

### CODE REVIEW

File: src/devices/schneider_electric.ts [hunk 1]
```diff
@@ -494,6 +525,7 @@ const definitions: Definition[] = [
         fromZigbee: [fz.fan],
         toZigbee: [tzLocal.fan_mode],
         exposes: [e.fan().withModes(['off', 'low', 'medium', 'high', 'on'])],
+        extend: [fanIndicatorMode(), fanIndicatorOrientation()],
```
Comment (lucasteligioridis) [inline src/devices/schneider_electric.ts:528]: I don't really understand how `extend` works, but I assume everything gets mapped correctly and you don't need to add any data/context into `fromZigbee` / `toZigbee`

Comment (Koenkk) [reply src/devices/schneider_electric.ts:528]: That's exactly how it works!

File: src/devices/schneider_electric.ts [hunk 2]
```diff
@@ -32,6 +32,37 @@ function indicatorMode(endpoint?: string) {
...
+function fanIndicatorMode() {
+    const description = 'Set Indicator Mode.';
+    return enumLookup({
+        name: 'indicator_mode',
+        lookup: {
+            'always_on': 3,
+            'on_with_timeout_but_as_locator': 4,
+            'on_with_timeout': 5,
+        },
+        cluster: 'clipsalWiserFanSwitchConfigurationCluster',
```
Comment (lucasteligioridis) [inline src/devices/schneider_electric.ts:44]: @Koenkk i need to make another PR this name was changed. I'll submit one shortly.

**Comment by lucasteligioridis** [comment]
@Koenkk https://github.com/Koenkk/zigbee-herdsman-converters/pull/7138
Please ensure this is merged before releasing as this PR is broken with the wrong cluster name.

**Pull request merged**